libs/net/dal/Services/ContentService.cs

```diff
@@ -554,7 +554,7 @@ public Content UpdateStatusOnly(Content updated)
         original.Version = updated.Version;
         this.Context.ResetVersion(original);
 
-        return base.Update(original);
+        return base.UpdateAndSave(original);
```

Comment:
@kylgarmor We have to methods, one updates the data in the DB, the other does not.  `Update` does not save to the DB.  It provides a way to do multiple actions with one transaction.  Use `UpdateAndSave` to save to DB.  I should have caught this originally :(